Pod/Classes/TSBaseParser.swift

```diff
@@ -26,7 +26,8 @@ public class TSBaseParser {
     private var parsingPairs = [TSExpressionBlockPair]()
     
     public func attributedStringFromMarkdown(markdown: String) -> NSAttributedString? {
-        return attributedStringFromMarkdown(markdown, attributes: defaultAttributes)
+        let attributedString = attributedStringFromMarkdown(markdown, attributes: defaultAttributes)
+        return attributedString
```

Comment:
Was this change to make breakpoints easier? It's not a big deal but I like how it was before better.